test/assets/messages/IndexMessages.scala

```diff
@@ -27,4 +27,6 @@ object IndexMessages {
   val tab1 = "Late submission penalties"
 
   val subheading = "Late submission penalties"
+
+  val externalGuidanceLinkText = "Read the guidance about late submission penalties (opens in a new tab)"
```

Comment:
Shouldn't this be a direct reference to the messages / messages.cy files? i.e. "{index.guidance.link} + " "  + {site.opensInNewTab}"
(Probably a question to clarify my understanding btw)

Reply:
In my opinion, it depends on preference.

I like to take what the prototype says. Because the view itself is using the messages - therefore if we check what's in the messages - it will always pass.

By checking against the text - we ensure that what's in the messages file (and therefore on the page) is correct.

If that makes sense?

Reply:
Ahhh I missed the fact that there were test assets xD
All makes sense now!